### PR TITLE
Move read marker past invisible events

### PR DESCRIPTION
**Pull request by bwindels**
Doesn't really fix any issue, but a nice clean-up (I came across while looking at https://github.com/vector-im/riot-web/issues/9889) to keep it symmetrical with how read receipts are moved.

### CODE REVIEW

File: src/components/structures/TimelinePanel.js [hunk 1]
```diff
@@ -1178,11 +1168,11 @@ const TimelinePanel = React.createClass({
                 adjacentInvisibleEventCount = 0;
             }
 
-            const shouldIgnore = (ignoreEchoes && ev.status) || // local echo
+            const shouldIgnore = !!ev.status || // local echo
```
Comment (jryans) [inline src/components/structures/TimelinePanel.js]: Hmm, so I guess this change read **receipt** handling to now also ignore local echoes, but hopefully that's okay...?

Comment (bwindels) [reply src/components/structures/TimelinePanel.js]: Oh, good point. I assumed that `this.state.liveEvents` wouldn't include pending events, but looking at `_getEvents`, they do. Will fix!

Comment (bwindels) [reply src/components/structures/TimelinePanel.js]: Oh, my comment above doesn't matter, as with this change, we are always ignoring local echoes here as you pointed out. We indeed should never send a read receipt for an event that hasn't been sent yet, so yes, this should be ok.

Comment (jryans) [reply src/components/structures/TimelinePanel.js]: Right, thanks for looking!

**Pull request merged**